tests/functional/adapter/materialized_view_tests/test_materialized_view_changes.py

```diff
@@ -63,6 +65,9 @@ def change_config_via_replace(project, materialized_view):
             initial_model.replace(old_partition, new_partition)
             .replace("'my_base_table'", "'my_other_base_table'")
             .replace('cluster_by=["id", "value"]', 'cluster_by="id"')
+            .replace(
```

Comment:
should we also test going from allow_non_incremental_definition=False to allow_non_incremental_definition=True